specification/extendedlocation/resource-manager/readme.md

```diff
@@ -26,7 +26,16 @@ These are the global settings for the extendedLocation.
 
 ```yaml
 openapi-type: arm
-tag: package-2021-08-31-preview
+tag: package-preview-2024-09
```

Comment:
can we keep this consistent with the existing format? let's use `package-2024-09-15-preview`

Reply:
Done